acme.sh

```diff
@@ -2361,13 +2363,13 @@ _process() {
           _webroot="$_webroot,$wvalue"
         fi
         ;;
-    --dnssleep 
+    --dnssleep)          
         re='^[0-9]+$'
-        if ! [[ $2 =~ $re ]] ; then
```

Comment:
Do not use "[[",  it's bash only.   We need keep sh compatible.
Use "["  instead.